Code/Editor/CryEdit.cpp

```diff
@@ -583,13 +583,15 @@ class CEditCommandLineInfo
             {{"exec", "cfg file to run on startup, used for systems like automation", "exec"}, m_execFile},
             {{"rhi", "Command-line argument to force which rhi to use", "dummyString"}, dummyString },
             {{"rhi-device-validation", "Command-line argument to configure rhi validation", "dummyString"}, dummyString },
-            {{"exec_line", "command to run on startup, used for systems like automation", "exec_line"}, m_execLineCmd},
+            {{"exec_line", "command to run on startup, used for systems like automation", "exec_linuser-pathe"}, m_execLineCmd},
```

Comment:
This line was corrupted with the last push

Reply:
Ops! nice catch, fixed!